specification/k3po.control/SPEC.md

```diff
@@ -0,0 +1,195 @@
+# K3po Control Protocol
+
+## Abstract
+
+This document defines the K3po control protocol used to communicate with a remote K3po server.
```

Comment:
I was thinking "FINISH" was a command but now I see it is "FINISHED".  I might be dense but the only spot where I see this is 

>The protocol is made up of Commands and Events.  Commands are actions that can be triggered by an external entity.
 Events are triggered by observed actions or state changes and are sent back to the external entity.  The protocol 
 is formatted by

where the external entity thing is confusing.  External could be the thing under tests, again I suggest standardizing the terminology.